### ISSUE TITLE
Wrong files added to this repository

### ISSUE THREAD

**Issue by rbmetcalf** [glenco/glamer#12]
@FabienNugier In the last commit to the master branch you added some of your files that have nothing to do with configuring GLAMER.  Please remove this commit after your sure your files have been stored somewhere.

**Comment by FabienNugier** [comment]
Hi. Sorry for the trouble. Which commit is it ?

**Comment by rbmetcalf** [comment]
Your last commit.  This is a repository for the configuration files and not the code files.  It is not SLsimLib or CosmoLib.  Only in exceptional circumstances would you ever add a file to this repository.

**Comment by FabienNugier** [comment]
I see it now, bfa506c... I really don't know what I did there but it is a mistake. Dunno how I did that... it is not even in my current project. I used "git reset -hard 7763343" on my local version, but I cannot push that because of error :

> The following untracked working tree files would be overwritten by merge:
> 	projects/Fraiseasy/output/PlotTestSub.py
> 	projects/Fraiseasy/readme
> Please move or remove them before you can merge.

I read online that I should delete untracked files, but I am worried to make it worst...

**Comment by rbmetcalf** [comment]
Are you trying to merge?
Don't merge into the master.  And pull before you push.

**Comment by rbmetcalf** [comment]
Try this.  Checkout the branch you added files to.  Do a `git rm` to the files and then commit the results.

**Comment by rbmetcalf** [comment]
If you don't have any need for these files push it and I will fix it.

**Comment by FabienNugier** [comment]
Hi Ben, Thanks. Actually pushing it in the current stage is not possible (head of master is behind the last commit), but I do not need these files (I saved them on mercurial in my own projects, where they should have been). So I guess you can remove this last commit on your side. On my side it seems impossible to remove it because it would be in conflict with the repository.

**Comment by rbmetcalf** [comment]
OK.  I think this is cleared up now.

**Comment by FabienNugier** [comment]
Thanks a lot ! (sorry again for this stupid mistake I did).

**Comment by rbmetcalf** [comment]
Its ok.  I discovered I did a similar thing on my local copy.  It happens 
when you add a new file from the wrong directory.  Strictly speaking we shouldn't 
have repositories in the subdirectories of other repositories.

Before you pull you should erase the last few commits from your local repository.
`git reset --hard HEAD~1`
Do this until you don't see your commit with `git log`.  Then you can pull the 
new version.

**Comment by FabienNugier** [comment]
Cool ! Thanks a lot. It is now completely cleaned on my side and I pulled.
